package proto_ff;

import "yd_fieldoptions.proto";


message E_GmcommandGmcommand
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "命令索引"];
	optional string m_key = 2[(yd_fieldoptions.field_cname) = "命令关键字", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_paramcount = 3[(yd_fieldoptions.field_cname) = "参数数量"];
	optional string m_des = 4[(yd_fieldoptions.field_cname) = "描述", (yd_fieldoptions.field_bufsize) = 64];
}


message Sheet_GmcommandGmcommand
{
	repeated E_GmcommandGmcommand E_GmcommandGmcommand_List = 1[(yd_fieldoptions.field_arysize)=64];
}
